docs/examples/haproxy/config.toml

```diff
@@ -3,8 +3,9 @@ dockerURL = "unix:///var/run/docker.sock"
 
 [[extensions]]
 name = "haproxy"
-configPath = "/usr/local/etc/haproxy.cfg"
-pidPath = "/etc/haproxy.pid"
+configPath = "/usr/local/etc/haproxy/haproxy.cfg"
```

Comment:
Left it as `/usr/local/etc/haproxy/haproxy.cfg` as per the [haproxy Dockerfile](https://github.com/docker-library/haproxy/blob/master/1.6/Dockerfile#L32)